src/client/index.js

```diff
@@ -34,4 +35,19 @@ module.exports.YotiClient = class YotiClient {
   performAmlCheck(amlProfile) {
     return amlService.performAmlCheck(amlProfile, this.pem, this.applicationId);
   }
+
+  /**
+   * @desc given a dynamic attribute list request get a custom QR code denoted by the
+   * dynamic attribute list stated in the request.
+   * @param {*} dynamicAttributeListRequest is the request that contains the attribute list.
+   * @returns {Promise} containing a dynamic attribute list
+   */
+  getQRCodeLink(dynamicAttributeListRequest) {
+    // Build a dynamic service, that gets the service
```

Comment:
```suggestion
    // Build a dynamic service, that gets the dynamic attribute list
```
`dynamic attribute list` / `QR Code`?